Detectors/Upgrades/ALICE3/Passive/include/Alice3DetectorsPassive/Pipe.h

```diff
@@ -37,29 +38,33 @@ class Alice3Pipe : public Alice3PassiveBase
   /// Clone this object (used in MT mode only)
   FairModule* CloneModule() const override;
 
-  float getInnerRmin() const { return mBeInnerPipeRmax - mBeInnerPipeThick; }
-  float getInnerRmax() const { return mBeInnerPipeRmax; }
+  float getInnerRmin() const { return mBeInnerPipeRmin; }
+  float getInnerRmax() const { return mBeInnerPipeRmin + mBeInnerPipeThick; }
   float getInnerWidth() const { return mBeInnerPipeThick; }
   float getInnerDz() const { return mInnerIpHLength; }
 
-  float getOuterRmin() const { return mBeOuterPipeRmax - mBeOuterPipeThick; }
-  float getOuterRmax() const { return mBeOuterPipeRmax; }
+  float getOuterRmin() const { return mBeOuterPipeRmin; }
+  float getOuterRmax() const { return mBeOuterPipeRmin + mBeOuterPipeThick; }
   float getOuterWidth() const { return mBeOuterPipeThick; }
   float getOuterDz() const { return mOuterIpHLength; }
 
+  bool getIsAlone() const { return mIsTRKActivated; }
```

Comment:
Sorry for being picky, but can you change the `IsAlone` also here?

Reply:
You are right, I missed this line.